!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  disparity_weight: 0.0001
  eps: 18.822322
  epsilon: 3.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      disparity_weight: 0.0001
      eps: 18.822322
      epsilon: 3.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      instance_weight: 0.003131
      invalid_disparity: false
      median_join: false
      min_pts: 3
      name: Default
      pairwise: true
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      pblg: 0.04
      pgrav: 0.1
      pground: 0.33
      pground_given_nexist: 0.28
      pnexist_dis: 0.25
      pobject: 0.33
      pobject_given_nexist: 0.44
      pord: 0.2
      pout: 0.15
      pout_sky: 0.4
      prior_weight: 1.0
      psky: 0.33
      psky_given_nexist: 0.28
      range_objects_z: 10.2
      road_vdisparity_threshold: 0.2
      segmentation_weight: 4.7095
      sigma_camera_height: 0.05
      sigma_camera_tilt: 0.05
      sigma_disparity_ground: 2.0
      sigma_disparity_object: 1.0
      sigma_sky: 0.1
      size_filter: 25
      state: true
      type: ''
    state: []
  instance_weight: 0.003131
  invalid_disparity: false
  median_join: false
  min_pts: 3
  pairwise: true
  pblg: 0.04
  pgrav: 0.1
  pground: 0.33
  pground_given_nexist: 0.28
  pnexist_dis: 0.25
  pobject: 0.33
  pobject_given_nexist: 0.44
  pord: 0.2
  pout: 0.15
  pout_sky: 0.4
  prior_weight: 1.0
  psky: 0.33
  psky_given_nexist: 0.28
  range_objects_z: 10.2
  road_vdisparity_threshold: 0.2
  segmentation_weight: 4.7095
  sigma_camera_height: 0.05
  sigma_camera_tilt: 0.05
  sigma_disparity_ground: 2.0
  sigma_disparity_object: 1.0
  sigma_sky: 0.1
  size_filter: 25
state: []
